src/vstest.console/Processors/EnableLoggerArgumentProcessor.cs

```diff
@@ -154,25 +155,32 @@ public void Initialize(string argument)
 
                 if (parseSucceeded)
                 {
-                    // First assume the logger is specified by URI. If that fails try with friendly name.
-                    try
+                    if (loggerIdentifier == "console")
                     {
-                        this.AddLoggerByUri(loggerIdentifier, parameters);
+                        this.loggerManager.AddLogger(new ConsoleLogger(), ConsoleLogger.ExtensionUri, parameters);
                     }
-                    catch (CommandLineException)
+                    else
                     {
-                        string loggerUri;
-                        if (this.loggerManager.TryGetUriFromFriendlyName(loggerIdentifier, out loggerUri))
+                        // First assume the logger is specified by URI. If that fails try with friendly name.
```

Comment:
This doesn't seem intuitive; user won't specify `/logger:logger://trx/v1`, they will just say `/logger:trx`.